adoc/chapters/programming_interface.adoc

```diff
@@ -12075,7 +12075,8 @@ is the cast to [code]#size_t#.
 a@
 [source]
 ----
-range operatorOP(const range& lhs, const size_t& rhs)
+// Available only when std::is_integral_v<T> is true
```

Comment:
Should this be a constrain?

Reply:
"Available only when" is how we used to specify constraints.  We _should_ add a "Constraints" paragraph to the description of these operators, but I don't think it makes sense to do that while they're in the old format (i.e., in a table).